src/index.js

```diff
@@ -0,0 +1,207 @@
+import './style.css';
+import getMeals from './modules/getMeals.js';
+import { postComment, getComments } from './modules/commentsApi.js';
+import totalComments from './modules/countComments.js';
+import totalMeals from './modules/countMeals.js';
+import { postLike, getLikes } from './modules/likesApi.js';
+
+const getNumberOfLikes = (response, likes) => {
+  const mealLikes = new Map();
+
+  response.forEach((meal) => {
+    let num = 0;
+    for (let i = 0; i < likes.length; i += 1) {
+      if (meal.idMeal === likes[i].item_id) {
+        num = likes[i].likes;
+        break;
+      }
+    }
+    mealLikes.set(meal.idMeal, num);
+  });
+  return mealLikes;
+};
+
+const renderMeal = async () => {
+  const response = await getMeals();
+
+  // const displayMeals = async () => {
+
+  //     const totalMealsNumber = document.querySelector('.totalMeals');
+  //      const response = await getMeals();
+  //     const total = totalMeals(response);
+  //     if (!total) {
+  //       totalMealsNumber.innerHTML = 0;
+  //     } else {
+  //       totalMealsNumber.innerHTML = total;
+  //     }
+  //     displayMeals();
```

Comment:
- Kindly make sure there is no commented-out block of code to make your project look professional :recycle: